packages/office-ui-fabric-react/src/components/Button/examples/Button.Split.Example.tsx

```diff
@@ -47,7 +47,11 @@ export class ButtonSplitCustomExample extends React.Component<IButtonProps, {}>
 
   public render() {
     let { disabled, checked } = this.props;
-    const style: IButtonStyles = { splitButtonMenuButton: { backgroundColor: 'white', width: '10px' }, splitButtonMenuIcon: { fontSize: '7px' } };
+    const style: IButtonStyles = {
```

Comment:
It is really not ideal to have any styles anywhere other than in class names or styles files. Is there any good reason this is nested in the component?

Reply:
Yeah agreed, I was just following the pattern here assuming it was done so it can appear in the sample code snippet on the demo site. I can move it.